rocky/reports/templates/report_types_selection.html

```diff
@@ -44,6 +44,8 @@ <h2>{% translate "Available report types" %} ({{ report_types_choices|length }})
                     <form class="inline layout-wide"
                           method="get"
                           action="{% url 'report_setup_scan' organization.code %}">
+                        {% include "forms/report_form_fields.html" %}
```

Comment:
Why is this added? This doesn't seem to be directly about checking for future dates and I don't see any mention of it in the PR description.